src/backend/routes/auth.js

```diff
@@ -37,5 +37,24 @@ module.exports = (passport) => {
     })(req, res, next);
   });
 
+  router.get('/auth/google', passport.authenticate('google', { scope: 'profile email' }));
+  router.get('/auth/google/callback', (req, res, next) => {
+    passport.authenticate('google', (err, user, info) => {
+      if (err) {
+        // next(err);
+      } else if (!user) {
+        res.json(Object.assign({ success: false }), info);
```

Comment:
Should be:
res.json(Object.assign({ success: false }, info));
Difference is the closing parens. If this was copied from the auth/login code I wrote, then my code has the same mistake